build.gradle

```diff
@@ -26,12 +26,23 @@ buildscript {
 }
 
 plugins {
+    id 'com.gradle.build-scan' version '1.9'
     id "com.jfrog.bintray" version "1.7.3"
     id "com.jfrog.artifactory" version "4.4.10"
     id "nebula.optional-base" version "3.1.0"
     id "nebula.provided-base" version "3.1.0"
 }
 
+buildScan {
+    licenseAgreementUrl = 'https://gradle.com/terms-of-service'
+    licenseAgree = 'yes'
+    link 'GitHub', 'https://github.com/cbeust/testng/tree/master'
+    if (System.getenv('CI')) {
+        publishOnFailure()
+        tag 'CI'
```

Comment:
Maybe we can add a tag for the jvm version

Reply:
Forget this idea, it is already provided here https://scans.gradle.com/s/vm5gbe54vglpu#infrastructure